src/main/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEvent.java

```diff
@@ -212,35 +213,36 @@ private CaseData getCaseData(final CaseData caseData, final DssCaseData dssCaseD
         List<ListValue<DssMessage>> listValues = new ArrayList<>();
```

Comment:
Can we rename this list to be something like messagesList - as it makes it clearer